README.rst

```diff
@@ -344,8 +344,7 @@ Or add this decorator to your test if you use Schemathesis in your Python tests:
     def test_api(case):
         ...
 
-**NOTE**. Schemathesis does not support examples in individual properties that are specified inside Schema Object.
-But examples in `Parameter Object <https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#parameter-object>`_, `Media Type Object <https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#mediaTypeObject>`_ and `Schema Object <https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#schema-object>`_ are supported.
+**NOTE**. Schemathesis now supports examples in individual properties.
```

Comment:
Hm, probably we can remove “now”, since Readme context doesn’t imply a history of changes